domain/src/main/java/edu/nextstep/camp/calculator/domain/Generator.kt

```diff
@@ -0,0 +1,29 @@
+package edu.nextstep.camp.calculator.domain
+
+class Generator {
+    companion object {
```

Comment:
```suggestion
object Generator {
```
클래스 내 멤버 함수가 없다면, object를 활용하시는 것이 좋습니다. (이하 관련 내용 동일)

Reply:
Generator라는 이름은 굉장히 포괄적인 의미를 가지고 있어요.
무엇을 만들어내는지 잘 모르겠네요. 🤔 
수식과 관련한 내용이니, Expression 과 같은 이름으로 만들어보면 어떨까요?

Reply:
저 클래스 자체의 용도가 좀 애매해서 더 이름짓기가 힘든 것 같네요 ..
input값을 textArr로 담아주면서 valid를 체크하는 내용이 섞여있어서 
이를 분리해보았습니다
Generator를 CalculatorInputConverter로 바꾸어보았습니다

Reply:
결국에는 보여주기 쉬운 수식으로 변환하는 로직이니, Expression 혹은 Formula와 같은 이름이 적절하지 않을까 싶었습니다. 😅